frontend/src/App.js

```diff
@@ -26,6 +27,26 @@ const useStyles = makeStyles((theme) => ({
  */
 export default function App() {
   const classes = useStyles();
+  const matchDataRefreshRate = 5000;
+  const [matchStatus, setMatchStatus] = React.useState('Unmatched');
```

Comment:
Can "unmatched" and "matched" be pulled out to constants?

Reply:
Yes, pulled them out to constants